core/parentchain/block-importer/Cargo.toml

```diff
@@ -58,4 +58,6 @@ sgx = [
     "itp-stf-executor/sgx",
     "itp-types/sgx",
     "thiserror_sgx",
-]
\ No newline at end of file
+]
+# feature to export mock implementations, only to be used for dev-dependencies!
+mocks = []
```

Comment:
Added a new `mocks` feature to this crate in order to export the mocks when needed. The reason I didn't place this in our `test/mock` crate, is that we will have circular dependencies with that mocks crate if a crate A both provides a trait that is mocked (`mocks` -> `A`), but also has tests that needs mocks (`A` -> `mocks`). So I decided to try out this approach. Keep the mocks inside the same crate and export it with a feature.

Reply:
No we don't have them (yet). But just because we don't have very many crates with mocks and tests that use them. With this crate, a circular dependency would've been introduced.

Reply:
Yes, but we should have had them in the encointer-node. However, I think your solution is superior anyhow. 👍